ansible/group_vars/all

```diff
@@ -25,6 +25,9 @@ controller:
   port: 10001
   blackboxFraction: 0.10
 
+invokerHealth:
+  testAction: whisk.system/utils/date
```

Comment:
At the moment it is not possible to transfer the action itself on Kafka.
We'd need to implement this with
- a new message
- the existing ActivationMessage

In both cases this implementation would not represent the real Health status between putting the activation on Kafka and getting the active ack.
The reason is, that we cannot put user actions on Kafka, because they might be too large.
So we cannot be sure, if the invoker (and the path to it) is really healthy.

In addition: Each invoker starts with the state `Healthy`. This action is only needed, if an invoker turns `UnHealthy` or `Offline`.